src/assessment.js

```diff
@@ -17,9 +17,9 @@
 // new function have access to the value stored in the variable `laseQuantity`
 // on line 3?
 //
-//     `true` or `false`
+//     'true' or `false`
 //
-var q1 = /* your answer here */;
+var q1 = /* FALSE */;
```

Comment:
Sorry, the answer here is `true`. A line following `var fire = function fire(){};` will not be inside the `fire` function; as such it will remain in the `deathStar` scope.